clientContext.ts

```diff
@@ -1,25 +1,33 @@
 import React, {Context} from 'react';
 
 export interface ClientProps {
+  avalancheCenter: string;
```

Comment:
Oops, I also didn't notice this - IMHO we should be careful to put things in context as it would be really easy to accidentally not make a screen routable / deep-link-able if we build something that is pulling from context, not route state

Reply:
mmm interesting point! I guess we could pull the `avalancheCenter` prop out of context so that it can't be read from. the only thing context is allowing us to do is call `setAvalancheCenter` from a nested component.

I'll do that real quick right now.